### PR TITLE
toggle json logging via config

### PR DESCRIPTION
**Pull request by kazauwa**
- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [x] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [x] updated CHANGELOG.md

Proposal pull request for #653

### CODE REVIEW

File: config-example.yaml [hunk 1]
```diff
@@ -173,6 +173,7 @@ tls_cert_path: ""
 tls_key_path: ""
 
 log_level: info
+json_logs: false
```
Comment (kradalby) [inline config-example.yaml:176]: I think this should probably be something like:

```suggestion
log:
  format: json
```

Allowing us to expand in the future.

Level should be migrated to under that root, but that might be a future thing

Comment (kazauwa) [reply config-example.yaml:176]: I can migrate level as well, but this is a backward incompatible change, so we should be careful about it. Is there a guideline for these kinds of changes?

Comment (juanfont) [reply config-example.yaml:176]: We are still in a phase were we can afford performing not-that-breaking changes :)

Comment (kazauwa) [reply config-example.yaml:176]: Amazing!
I'm sorry for disappearing, was really busy these days. Hopefully will finish this MR this week.

**Review by juanfont — APPROVED** [review]

**Comment by juanfont** [comment]
Thanks!!

**Pull request merged**